pkg/operator/controller/extension/admission/admission.go

```diff
@@ -196,14 +200,24 @@ func (d *deployment) createOrUpdateAdmissionVirtualClusterResources(ctx context.
 			return err
 		}
 	}
+	namespace := virtualNamespace(extension)
+	registry := managedresources.NewRegistry(kubernetes.ShootScheme, kubernetes.ShootCodec, kubernetes.ShootSerializer)
```

Comment:
```suggestion
	registry := managedresources.NewRegistry(kubernetes.GardenScheme, kubernetes.GardenCodec, kubernetes.GardenSerializer)
```

Nit: Shouldn't make a difference in this case, since overlapping schemes, but matches the cluster in use.